cmd/meroxa/root/flink/list_test.go

```diff
@@ -0,0 +1,116 @@
+/*
+Copyright © 2022 Meroxa Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package flink
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"reflect"
+	"strings"
+	"testing"
+
+	"github.com/meroxa/cli/utils/display"
+
+	"github.com/golang/mock/gomock"
+	"github.com/meroxa/cli/log"
+	"github.com/meroxa/cli/utils"
+	"github.com/meroxa/meroxa-go/pkg/meroxa"
+	"github.com/meroxa/meroxa-go/pkg/mock"
+	"github.com/stretchr/testify/assert"
+)
+
+func getFlinkJobs() []*meroxa.FlinkJob {
+	var flinkJobs []*meroxa.FlinkJob
+	f := utils.GenerateFlinkJob()
+	return append(flinkJobs, &f)
+}
+
+func TestListFlinkJobsExecution(t *testing.T) {
+	ctx := context.Background()
+	ctrl := gomock.NewController(t)
+	client := mock.NewMockClient(ctrl)
+	logger := log.NewTestLogger()
+
+	flinkJobs := append(getFlinkJobs())
```

Comment:
```suggestion
	flinkJobs := getFlinkJobs()
```
you shouldn't need append here